requirements.txt

```diff
@@ -0,0 +1,10 @@
+attrs==21.4.0
+iniconfig==1.1.1
+numpy==1.22.3
+packaging==21.3
+pluggy==1.0.0
+py==1.11.0
+pyparsing==3.0.7
+pytest==7.1.1
+tomli==2.0.1
+typing==3.7.4.3
```

Comment:
There are standard library and no need to include it. But it's good to have a requirements.txt.

Reply:
Yeah, that's true. Since I generated requirements via `pip freeze` they stayed.